include/swift/AST/ASTContext.h

```diff
@@ -990,10 +990,10 @@ class ASTContext final {
     return getSwiftAvailability(5, 10);
   }
 
-  /// Get the runtime availability of features introduced in the Swift 5.11
+  /// Get the runtime availability of features introduced in the Swift 6.0
   /// compiler for the target platform.
   inline AvailabilityContext getSwift511Availability() const {
```

Comment:
Nit: We should rename `getSwift511Availability()` to `getSwift60Availability()`. That's cosmetic though and can be done in a follow-up PR.

Reply:
Updated, thanks!